Pod/Classes/Optional Data Sources/VOKFetchedResultsDataSource.h

```diff
@@ -21,22 +21,21 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface VOKFetchedResultsDataSource : NSObject <NSFetchedResultsControllerDelegate, UITableViewDelegate, UITableViewDataSource>
 
-@property (readonly) Class managedObjectClass;
-@property (weak, readonly) UITableView *tableView;
-@property (weak, readonly) NSManagedObjectContext *managedObjectContext;
+@property (nonatomic, assign, readonly) Class managedObjectClass;
+@property (nonatomic, weak, readonly) UITableView *tableView;
 
-@property (weak) id<VOKFetchedResultsDataSourceDelegate> delegate;
+@property (nonatomic, weak) id<VOKFetchedResultsDataSourceDelegate> delegate;
 
-@property (strong, readonly) NSFetchedResultsController *fetchedResultsController;
+@property (nonatomic, strong, readonly) NSFetchedResultsController *fetchedResultsController;
```

Comment:
I guess the question was "Why did it ever work?"
but if found "...if your property is readonly AND you implement your own getter method, then an instance variable is not synthesized." So this is working correctly. Nevermind!